Cargo.toml

```
@@ -31,6 +31,7 @@ gzip = ["dep:flate2"]
 brotli = ["dep:brotli-decompressor"]
 charset = ["dep:encoding_rs"]
 json = ["dep:serde", "dep:serde_json", "cookie_store?/serde_json"]
+digest-auth = ["digest_auth"]
```

I think it would me more correct/consistent to say:
```suggestion
digest-auth = ["dep:digest_auth"]
```